corehq/tests/nose.py

```diff
@@ -209,26 +213,30 @@ def setup(self):
             databases = {app_name: uri for app_name, uri in databases}
         self.apps = [self.verify_test_db(*item) for item in databases.items()]
 
-        if self.skip_setup_for_reuse_db:
-            from django.db import connections
-            old_names = []
-            for connection in connections.all():
-                db = connection.settings_dict
-                assert db["NAME"].startswith(TEST_DATABASE_PREFIX), db["NAME"]
-                try:
-                    connection.ensure_connection()
-                except OperationalError:
-                    break  # cannot connect; resume normal setup
-                old_names.append((connection, db["NAME"], True))
-            else:
-                self.old_names = old_names, []
```

Comment:
Looks like you removed this bit of logic, which is used in [django-nose db teardown](https://github.com/millerdev/django-nose/blob/fast-first/django_nose/plugin.py#L434) and is important for `REUSE_DB=teardown`. I should have documented that in a comment.